controllers/networking-calico/charts/internal/calico/templates/node/configmap-calico-config.yaml

```
@@ -16,7 +16,7 @@ data:
   # Configure the Calico backend to use.
   calico_backend: "{{ .Values.config.backend }}"
   # Configure the MTU to use
-  veth_mtu: "1440"
+  vethMTU: "{{ .Values.config.vethMTU }}"
```

```suggestion
  veth_mtu: "{{ .Values.config.vethMTU }}"
```

This is the calico configmap, so I think at least the key here must be `veth_mtu`, right?